verify_schedule.py

```
@@ -0,0 +1,92 @@
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        handlers=[
+            logging.FileHandler("discrepancy.log"),
+            logging.StreamHandler(),
+        ],
+    )
+
+
+def get_real_time_data():
+    """Fetches the real-time data from the API."""
+    try:
+        response = requests.get("http://localhost:5000/station/now")
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)